spec/OamLog+config.json

```diff
@@ -1377,7 +1377,7 @@
                 "tcp-client-interface-1-0:tcp-client-interface-pac": {
                   
                   "tcp-client-interface-configuration": {
```

Comment:
line 1378 : somehow a line space has been added to this file. Kindly requesting you to remove this space.

Reply:
line space deleted.